packages/paste-website/src/pages/components/sidebar-navigation/index.mdx

```diff
@@ -0,0 +1,849 @@
+- A `selected` `SidebarNavigationItem` is designated as the "current page" via `aria-current="page"`.
+
+## Examples
+
+### Flat
+
+In its most basic form, the Sidebar Navigation can be utilized to create a straightforward, flat list of navigation items. This is ideal for smaller applications or single-product use cases.
+
+Each `SidebarNavigationItem` should be provided with an `href`, a valid URL that the user will navigate to upon interaction. This facilitates opening links in new tabs via cmd/ctrl + clicking or right-clicking on the item, ensuring proper keyboard navigation for keyboard-only users.
+
+The `SidebarNavigationItem` can be decorated with an icon by setting the icon property value to any valid [Paste Icon](/components/icons).
+
+To indicate the current page, the `selected` property should be set on the item that represents the user's current location in the application.
+
+<LivePreview
+  scope={{
+    Box,
+    SidebarBody,
+    SidebarNavigation,
+    SidebarNavigationItem,
+    ProductHomeIcon,
+    ProductContactCenterAdminIcon,
+    ProductContactCenterTasksIcon,
+    ProductContactCenterTeamsIcon,
+    ProductContactCenterQueuesIcon,
+    ProductPrivacyIcon,
+    ProductUsageIcon,
+  }}
+  language="jsx"
+>
+  {`<Box backgroundColor="colorBackgroundInverse">
+  <SidebarBody>
+    <SidebarNavigation>
+      <SidebarNavigationItem
+        href="https://google.com"
+        selected
+        icon={<ProductContactCenterAdminIcon decorative={false} title="Admin" />}
+      >
+        Admin
+      </SidebarNavigationItem>
+      <SidebarNavigationItem
+        href="https://google.com"
+        icon={<ProductContactCenterTasksIcon decorative={false} title="Agent dashboard" />}
+      >
+        Agent dashboard
+      </SidebarNavigationItem>
+      <SidebarNavigationItem
+        href="https://google.com"
+        icon={<ProductContactCenterTeamsIcon decorative={false} title="Teams view" />}
+      >
+        Teams view
+      </SidebarNavigationItem>
+      <SidebarNavigationItem
+        href="https://google.com"
+        icon={<ProductContactCenterQueuesIcon decorative={false} title="Queue stats" />}
+      >
+        Queue stats
+      </SidebarNavigationItem>
+      <SidebarNavigationItem
+        href="https://google.com"
+        icon={<ProductPrivacyIcon decorative={false} title="Privacy" />}
+      >
+        Privacy
+      </SidebarNavigationItem>
+      <SidebarNavigationItem
+        href="https://google.com"
+        icon={<ProductUsageIcon decorative={false} title="Insights" />}
+      >
+        Insights
+      </SidebarNavigationItem>
+    </SidebarNavigation>
+  </SidebarBody>
+</Box>`}
+</LivePreview>
+
+### Hierarchical
+
+Sidebar Navigation also supports the creation of hierarchical information architectures using nested disclosures. You can establish "groups" of navigation sections by deploying the `SidebarNavigationSeparator` between groups of items.
```

Comment:
```suggestion
Sidebar Navigation also supports hierarchical information architectures with nested navigation items. You can establish "groups" of navigation sections by using `SidebarNavigationSeparator` between groups of items.
```

Reply:
I think there's a simpler way to explain this Hierarchical section overall, but I'll just take a look over again when I reread through all the docs.